stdeb.cfg

```diff
@@ -1,5 +1,6 @@
 [colcon-common-extensions]
 No-Python2:
 Depends3: python3-colcon-argcomplete, python3-colcon-bash, python3-colcon-cd, python3-colcon-cmake, python3-colcon-core, python3-colcon-defaults, python3-colcon-devtools, python3-colcon-library-path, python3-colcon-metadata, python3-colcon-notification, python3-colcon-output, python3-colcon-package-information, python3-colcon-package-selection, python3-colcon-parallel-executor, python3-colcon-powershell, python3-colcon-python-setup-py, python3-colcon-recursive-crawl, python3-colcon-ros, python3-colcon-test-result, python3-colcon-zsh
+Recommends3: python3-colcon-override-check
```

Comment:
Not all config values have a `3`-suffixed variation but Recommends, Suggests, Provides, and Replaces all do.

Since these packages are python3-only and include the `No-Python2` config directive using the 3-suffixed config values is unnecessary but not worth changing.